features/other_a/languages.feature

```diff
@@ -1,5 +1,32 @@
 @works
 Feature: Languages
+
+  Scenario: Add a language and check abuse and support available
```

Comment:
Right, sorry. I mean in the admin_languages.feature you want a test for an admin adding a language:

```
Scenario: An admin can add a language

Given basic languages
  And I am logged in as an admin
When I go to the languages page
  And I follow "Add a Language"
  And I fill in "Name" with "Klingon"
  And I fill in "Abbreviation" with "tlh"
  And I press "Create Language"
Then I should see "Language was successfully added."
  And I should see "The Archive supports these languages"
  And I should see "Klingon"
```

And another for an admin adding support for a language:

```
Scenario: Adding Support for a language

  Given the following language exists
    | name           | short |
    | Francais       | fr    |
  When I am logged in as an admin
    And I go to the languages page
    And I follow "Edit"
    And I check "Support available"
    And I press "Update Language"
  When I follow "Technical Support and Feedback"
    And I should see "Francais" within "select#feedback_language"
```